incubator/crate/templates/statefulset.yaml

```
@@ -0,0 +1,98 @@
+apiVersion: "apps/v1"
+kind: StatefulSet
+metadata:
+  name: crate
+spec:
+  serviceName: "crate-db"
+  replicas: {{ .Values.crate.replicas }}
+  selector:
+    matchLabels:
+      app: {{ .Values.app }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.app }}
+        app.kubernetes.io/name: crate-deployment
+        app.kubernetes.io/part-of: crate-db
+        app.kubernetes.io/version: {{ .Values.image.tag }}
+        app.kubernetes.io/managed-by: helm
+      annotations:
+        pod.alpha.kubernetes.io/initialized: "true"
+    spec:
+      initContainers:
+      - name: init-sysctl
+        image: busybox
+        imagePullPolicy: IfNotPresent
+        command: ["sysctl", "-w", "vm.max_map_count=262144"]
+        securityContext:
+          privileged: true
+      containers:
+      - name: crate
+        image: {{ .Values.image.name }}:{{ .Values.image.tag }}
```

```suggestion
        image: crate:{{ .Values.image.tag }}
```

Per https://github.com/helm/charts/pull/16318/files#r321134391